src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,30 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val greeters = listOf(
```

Comment:
Interesting solution, and valid in this particular case.
However, please note that wrapping 'Hello-objects' in list has some limitations like less flexibility, because greeters are incapsulated in the list and as a result there is nested loop in row 30.
Please consider this in the future with similar situations